src/routes/landing/components/FilterableLocationSelector.js

```diff
@@ -25,10 +25,11 @@ class FilterableLocationSelector extends React.Component {
   render () {
     return (
       <div className={style.topSpacing}>
-        <Heading/>
+        <Heading />
```

Comment:
Avoid refactors in other modules. We have separate tasks for this :) No problem in this case as I did not start the landing task yet.

Reply:
I couldn't hold it back anymore :(